TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -253,18 +267,51 @@ public async Task<CommandResult> Pokedex(CommandContext context)
                     ResponseTarget = ResponseTarget.WhisperIfLong
                 };
             }
+            else if (mode.Equals(PokedexModeKantoooooooo))
+            {
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == Generation.Gen1);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                };
```

Comment:
could this be done without duplicating code of the other mode branch?

Reply:
Removed the code for now, but I'll try to make this one work when I find the motivation

>don't make the easter egg mandatory. I believe some people might actually still parse bot output instead of using the API
> you can try to make the last letter for all regions repeatable
> johtooo -> JOHTOOO :dog: